articles/quantum-QR-TypeModel.md

```diff
@@ -329,3 +329,6 @@ so `Controlled(Rz)` has type
 For example, `((Controlled(Rz))(controls, (0.1, target))` would be 
 a valid invocation of `Controlled(Rz)`.
 
+As another example, `CNOT(control, target)` analogue can be implemented as `(Controlled(X))([control], target)`. 
```

Comment:
I wouldn't call it an "analogue" (here or on the following line), Controlled X with one or two control qubits literally are CNOT and CCNOT. I'd remove "analogue" and say just "can be implemented".

Reply:
@tcNickolas both fixes have been done.